datanode/gateway/graphql/liquidity_provider_resolver.go

```diff
@@ -18,3 +18,14 @@ func (r *liquidityProviderResolver) FeeShare(_ context.Context, obj *v2.Liquidit
 		VirtualStake:          obj.FeeShare.VirtualStake,
 	}, nil
 }
+
+func (r *liquidityProviderResolver) SLA(_ context.Context, obj *v2.LiquidityProvider) (*LiquidityProviderSLA, error) {
+	return &LiquidityProviderSLA{
+		Party:                            &vegapb.Party{Id: obj.PartyId},
+		CurrentEpochFractionOfTimeOnBook: obj.Sla.CurrentEpochFractionOfTimeOnBook,
+		LastEpochFractionOfTimeOnBook:    obj.Sla.LastEpochFractionOfTimeOnBook,
+		LastEpochFeePenalty:              obj.Sla.LastEpochFeePenalty,
+		LastEpochBondPenalty:             obj.Sla.LastEpochBondPenalty,
+		HysteresisPeriodFeePenalties:     obj.Sla.HysteresisPeriodFeePenalties,
+	}, nil
+}
```

Comment:
You can simplify this by defining a mapping between the GQL LiquidityProviderSLA object and the proto in the gqlgen.yml and just probably return `obj.Sla` for this resolver.